.github/workflows/deploy.yml

```diff
@@ -53,3 +54,52 @@ jobs:
         run: |
           chmod +x ./scripts/deploy.sh
           ./scripts/deploy.sh
+
+  deploy-firebase:
+    name: Build, Test, and Deploy Firebase
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        working-directory: permission-portal-cloud-functions/functions
+        run: npm install
+
+      - name: Initialize config files
+        working-directory: permission-portal-cloud-functions/functions
+        run: npm run init        
+
+      - name: Build and Deploy to Firebase Test Infra
```

Comment:
Redundant, we can assume that if a branch has been merged into master or dev, tests have already run and passed (in the other workflow).